lib/src/data/repositories/movie_repository.dart

```diff
@@ -2,14 +2,36 @@ import 'dart:convert';
 
 import 'package:flutter/services.dart';
 
-import '../../domain/repository/i_repository.dart';
-import '../../domain/entity/movies_model.dart';
+import '../../core/util/string_constants.dart';
+import '../../domain/entity/movie.dart';
+import '../../domain/entity/response.dart';
+import '../../domain/repository/i_movie_repository.dart';
+import '../datasource/remote/api_service.dart';
+import '../models/movie_response.dart';
+
+class MovieRepository implements IMovieRepository<Movie> {
+  final ApiService service;
+  static const String baseUrl = StringConstants.baseUrl + 'movie/';
```

Comment:
You can made a get method that returns you the baseUrl like
String get baseUrl => '${StringConstants.baseUrl}movie/';

Reply:
done